### ISSUE TITLE
Exception on Project Create

### ISSUE THREAD

**Issue by Rennerdo30** [maikebing/GitLab.VisualStudio#52]
### Installed product versions
- Visual Studio: 2017 15.9.11
- This extension: 1.0.183

### Description
An exception occours, if a solution is loaded and the create project button is pressed.

### Steps to recreate
1. Load solution
2. Switch to Team Explorer and press Create

### Current behavior
Exception is thrown:
```
[GitLab for Visual Studio  Handled Exception 04:17:33 ] Diagnostics mode caught and marked as handled the following DispatcherUnhandledException raised in Visual Studio: System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   bei GitLab.VisualStudio.UI.ViewModels.CreateViewModel.LoadResources()
   bei GitLab.VisualStudio.UI.ViewModels.CreateViewModel..ctor(IDialog dialog, IGitService git, IMessenger messenger, IShellService shell, IStorage storage, IWebService web)
   bei GitLab.VisualStudio.Services.ViewFactory.GetView[T](ViewTypes type)
   bei GitLab.TeamFoundation.ViewModels.ConnectSectionViewModel.OnCreate()
   bei GitLab.VisualStudio.Shared.Helpers.Commands.DelegateCommand.<>c__DisplayClass5_0.<.ctor>b__0(Object o)
   bei GitLab.VisualStudio.Shared.Helpers.Commands.DelegateCommand.Execute(Object parameter)
   bei GitLab.VisualStudio.Shared.Helpers.Commands.DelegateCommand.System.Windows.Input.ICommand.Execute(Object parameter)
   bei MS.Internal.Commands.CommandHelpers.CriticalExecuteCommandSource(ICommandSource commandSource, Boolean userInitiated)
   bei System.Windows.Documents.Hyperlink.OnClick()
   bei System.Windows.Documents.Hyperlink.DispatchNavigation(Object sender)
   bei System.Windows.Documents.Hyperlink.DoUserInitiatedNavigation(Object sender)
   bei System.Windows.Documents.Hyperlink.OnMouseLeftButtonUp(Object sender, MouseButtonEventArgs e)
   bei System.Windows.Documents.Hyperlink.OnMouseLeftButtonUp(MouseButtonEventArgs e)
   bei System.Windows.UIElement.OnMouseLeftButtonUpThunk(Object sender, MouseButtonEventArgs e)
   bei System.Windows.Input.MouseButtonEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   bei System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   bei System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   bei System.Windows.UIElement.OnMouseUpThunk(Object sender, MouseButtonEventArgs e)
   bei System.Windows.Input.MouseButtonEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   bei System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   bei System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   bei System.Windows.ContentElement.RaiseTrustedEvent(RoutedEventArgs args)
   bei System.Windows.Input.InputManager.ProcessStagingArea()
   bei System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   bei System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   bei System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   bei System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

```

### Expected behavior
Dialog opens. Project can be created.

**Comment by maikebing** [comment]
I haven't been able to reproduce.

**Comment by Rennerdo30** [comment]
I've found the bug.
The problem was, that you're code expects that an `*.gitlab4vs` for the remote of the current project exits.
This is not always the case, if you have multiple GitLab's.

My fix in PR #56 is more likely an workaround, because this will fallback to `gitlab.com.gitlab4vs` if `*.gitlab4vs does` not exist.
